Makefile

```
@@ -45,6 +46,14 @@ go: base-image-go
 	  -I/opt/include -I/googleapis -I/defs/protos \
 	  --go_opt=module=github.com/sigstore/protobuf-specs/gen/pb-go --go_out=/defs/gen/pb-go ${PROTOS}
 
+# an image on ghcr for generating defintions for sigstore services
+services-image:
```

```suggestion
services-image: go
```
should we make the dependency more explicit?